ksqldb-rest-model/src/main/java/io/confluent/ksql/rest/entity/KafkaStreamsStateCount.java

```diff
@@ -0,0 +1,130 @@
+/*
+ * Copyright 2020 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"; you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.ksql.rest.entity;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonValue;
+import io.confluent.ksql.util.KsqlConstants;
+import io.confluent.ksql.util.KsqlException;
+
+import java.util.Comparator;
+import java.util.Map;
+import java.util.Objects;
+import java.util.TreeMap;
+
+import org.apache.kafka.streams.KafkaStreams;
+
+/**
+ * Used to keep track of a the state of KafkaStreams application
+ * across multiple servers. Used in {@link RunningQuery}.
+ */
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class KafkaStreamsStateCount {
+  
+  // Use a TreeMap so toString() will always return the same string
+  private final TreeMap<KafkaStreams.State, Integer> state;
+
+  public KafkaStreamsStateCount() {
+    this.state = returnTreeMap();
+  }
+
+  @JsonCreator
+  public KafkaStreamsStateCount(final String serializedPair) {
+    final String [] parts = serializedPair.split(",");
```

Comment:
for SerDe of the maps, I would recommend using:
```java
    Map<String, String> pairs = Splitter.on(",").withKeyValueSeparator(":").split(serialized);
    state =  pairs.entrySet().stream()
        .collect(Collectors.toMap(
            e -> KafkaStreams.State.valueOf(e.getKey()),
            e -> Integer.parseInt(e.getValue()),
            (v1, v2) -> {throw new IllegalStateException("duplicate keys")},
            TreeMap::new
        ));
```
And you can use the correspond method to serialize: `Joiner.on(",").withKeyValueSeparator(":").join(values);`

Reply:
done